publisher-advertising.md

```diff
@@ -0,0 +1,112 @@
+| `params` | array of JSON objects | false | An optional list of query parameter key/values that must be included in requests |
+
+Contact information should follow the given format:
+
+| field name | type | required | description |
+| --- | --- | --- | ---|
+| `name` | string | true | Name of contact |
+| `email` | string | true | Email address of contact |
+
+Header and Query param information should have the following format:
+
+| field name | type | required | description |
+| --- | --- | --- | ---|
+| `name` | string | true | Name of required header/parameter |
+| `email` | string true | value of required header/parameter |
+
+An example is given below:
+
+```json
+{
+  "name": "SMART Slot Publisher Specification Repository",
+  "organization": "SMART",
+  "synthetic": true,
+  "baseURL": "https://raw.githubusercontent.com/smart-on-fhir/smart-scheduling-links/master/examples",
+  "contacts": [
+    {
+      "name": "Nick Robison",
+      "email": "nicholas.a.robison@omb.eop.gov"
+    },
+    {
+      "name": "Josh Mandel",
+      "email": "jmandel@gmail.com"
+    }
+  ],
+  "headers": [
+    {
+      "name": "CLIENT_ID",
+      "value": "test-client-id"
+    }
+  ],
+  "params": [
+    {
+      "name": "clientid",
+      "value": "test-client-id"
+    }
+  ]
+}
+```
+
+### Submitting an endpoint to the repository
+
+Endpoint submission is handled through Github Pull Requests (PR).
+Each PR will be reviewed by project administrators to ensure upstream data is correct and from an authentic publisher.
+
+Users may submit a PR by the following process:
+
+1. Create a [Fork](https://github.com/smart-on-fhir/smart-scheduling-links/fork) of the repository in order to make changes.
+1. Create a new `.json` file in the [publisher-repository](/publisher-repository) directory.
+   
+    A JSON schema file is [provided](bin/upstream-validator/schema.json) which can facilitate data creation.
+1. Fill out the required information in the format given above.
+1. Push changes to your fork
+1. Create a [pull request](https://github.com/smart-on-fhir/smart-scheduling-links/compare) against the repository.
+
+    Please include descriptive information that will allow the project administrators to verify the authenticity of the data and the request.
+    Administrators may request additional information and/or verification before accepting the proposed addition. 
+
+## Schema.org definition
+
+Organizations may choose to publish their endpoints by including an additional _schema.org_ `Service` description to their home page.
+A sample service definition is included below:
+
+```html
+<script type="application/ld+json">
+{
+  "@context": "https://schema.org/",
+  "@type": "Service",
+  "provider": {
+    "@type": "Organization",
+    "name": "Carbon Health"
+  },
+  "serviceType": "https://github.com/smart-on-fhir/smart-scheduling-links",
+  "url": "https://api.carbonhealth.com/hib/publicVaccination/$bulk-publish"
```

Comment:
@jmandel Should this follow the same format as `baseURL`,(e.g. exclude the $bulk-publish component) not sure if there's a convention already.